### DISCUSSION TITLE
Is it possible to train with multiple GPUs?

### DISCUSSION THREAD

**Discussion by elozano98** [explosion/spaCy#8630] [Help: Other Questions]
# Train with multiple GPUs
I am training my transformer pipeline using my GPU:
```bash
$ python -m spacy train ./config/config.cfg --output ./models --gpu-id 0
```
Since I have a lot of data, the training process is not fast enough, so I would like to know if there is an option to enable more than one GPU when training.

Thank you!

**Comment by MightyGoldenOctopus** [comment]
Take a look to distributed training capabilities through Ray with SpaCy: https://spacy.io/usage/training#parallel-training

**Comment by polm** [comment]
Multi-GPU support is still very much a work in progress. Take a look at https://github.com/explosion/spaCy/issues/8093 for a recent thread on the subject.

**Reply by elozano98** [reply]
I've read something about using [spacy-ray](https://github.com/explosion/spacy-ray) to configure workers so that each one is associated with one particular GPU. Is it possible?

**Reply by polm** [reply]
That's the approach they took in that thread, which I think had some issues. That's everything I know about the subject, sorry.
